mbed_lib.json

```diff
@@ -18,7 +18,7 @@
             "value": "DEFAULT_MAX_APPLICATION_SIZE"
         },
         "trace": {
-            "help": "Show boot status and progress on serial output. Use USE_DIRECT_SERIAL_OUTPUT to save space.",
+            "help": "Show boot status and progress on serial output. Use USE_DIRECT_SERIAL_OUTPUT for minimal prints to save space. Use `null` to disable traces",
```

Comment:
Shouldn't be 'null' added to the 'accepted values'?

Reply:
`null` means that `MBED_CONF_MBED_BOOTLOADER_TRACE` macro won't be defined at all. It is not part of the values, but undefine to the macro